src/components/SocialMedia/SocialMedia.css

```diff
@@ -1,23 +1,23 @@
 .SocialMediaIcon {
-    margin: 0 1em;
-    transition: 0.3s ease-in-out;
-    display: inline-block;
+  margin: 0 1em;
+  transition: 0.3s ease-in-out;
+  display: inline-block;
 }
 
 .SocialMediaIcon-FooterColor {
-    color: #282A36;
+  color: #d1d3e0;
 }
 
 .SocialMediaIcon-DefaultColor {
-    color: var(--primary-green);
+  color: var(--primary-green);
 }
 
 .SocialMediaIcon:hover {
-    transform: translateY(-0.25em);
-    color: white;
-    box-shadow: 0 0.5em 0.5em -0.4em white; /* offset-x | offset-y | blur-radius | spread-radius | color*/
+  transform: translateY(-0.25em);
+  color: white;
+  box-shadow: 0 0.5em 0.5em -0.4em white; /* offset-x | offset-y | blur-radius | spread-radius | color*/
```

Comment:
![image](https://user-images.githubusercontent.com/45266424/120916454-ee2fb580-c666-11eb-8067-b20be45fd692.png)
I don't really like the box shadow on these- it looks too jarring. We either need to improve the effect, remove it, or replace it with something else entirely